plugins/modules/swim_intent.py

```diff
@@ -499,32 +504,31 @@ def get_image_id(self, name):
             function='get_software_image_details',
             params={"image_name": name},
         )
-
-        self.log(str(image_response))
-
+        self.log("Received API response from 'get_software_image_details': {0}".format(str(image_response)), "DEBUG")
         image_list = image_response.get("response")
+
         if (len(image_list) == 1):
             image_id = image_list[0].get("imageUuid")
-            self.log("Image Id: " + str(image_id))
+            self.log("SWIM image {0} having the Id: {1}".format(name, image_id), "INFO")
         else:
-            error_message = "Image {0} not found".format(name)
-            self.log(error_message)
-            self.module.fail_json(msg="Image not found", response=image_response)
+            error_message = "SWIM image {0} not found".format(name)
```

Comment:
error_message = "SWIM image '{0}' could not be found".format(name)

Reply:
added.